estate/models/estate_property.py

```diff
@@ -0,0 +1,34 @@
+from odoo import models, fields
+from dateutil.relativedelta import relativedelta
+
+
+class EstateModel(models.Model):
```

Comment:
try to keep python class name consistent with the `_name` as it would be used in future
```suggestion
class EstateProperty(models.Model):
```

Reply:
Fixed.